server/bin/pbench-backup-tarballs.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import sh
+import glob
+import base64
+import signal
+import configtools
+import subprocess
+from argparse import ArgumentParser
+
+# Correct the first path to be the "lib" directory to make the
+# pbench-backup-to-s3 script simple, allowing the imports from the
+# backups3 library to work cleanly.
+first_path = sys.path[0]
+assert first_path.endswith('/bin')
+lib_path = os.path.join(os.path.dirname(first_path), 'lib')
+sys.path[0] = lib_path
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger
+from s3backup import S3Config
+
+_NAME_    = "pbench-backup-tarballs"
+
+# The link source and destination for this operation of this script.
+_linksrc = "TO-BACKUP"
+_linkdest = "BACKED-UP"
+
+# Global logger for the module, setup in main()
+_logger = None
+
+def rsync_archive_backup(config, report):
```

Comment:
Drop this routine in favor of adding a simple copy of the tar ball and its `.md5` just before sending to S3.

We are doing this because we no longer need to review the entire `ARCHIVE` tree, we only need to copy what is linked in `TO-BACKUP`, which should always be a new file that won't exist in the backup tree (or S3), so rsync is not needed.